tooling/src/hypothesistooling/__main__.py

```diff
@@ -403,31 +403,65 @@ def update_django_versions():
 
 
 def update_pyodide_versions():
+
+    def version_tuple(v: str) -> tuple[int, int, int]:
+        return tuple(int(x) for x in v.split("."))  # type: ignore
```

Comment:
I kept this as a nested function as it's only a one-liner; happy to move it outside as a private function if you suggest.